Document XSLTProcessor.* (#25563)
mdn/content: files/en-us/web/api/xsltprocessor/reset/index.md

---
title: XSLTProcessor.reset()
slug: Web/API/XSLTProcessor/reset
page-type: web-api-instance-method
browser-compat: api.XSLTProcessor.reset 
---

{{APIRef("XSLT")}}

> The `reset()` method of the {{domxref("XSLTProcessor")}} interface removes all parameters (`<xsl:param>`) and the XSLT stylesheet from the processor. The `XSLTProcessor` will then be in its original state, like after its creation.esheet.

Reviewer: ```suggestion
The `reset()` method of the {{domxref("XSLTProcessor")}} interface removes all parameters (`<xsl:param>`) and the XSLT stylesheet from the processor. The `XSLTProcessor` will then be in its original state when it was created.
```
